packages/site/pages/components/note.js

```diff
@@ -0,0 +1,240 @@
+        linking straight to the content.
+      </P>
+      <Example.React
+        themeToggle
+        includes={{ Avatar, Note, Icon }}
+        codes={[
+          `
+<Note
+  actionBar={[
+    <Note.Action icon={<Icon id={Icon.ids.bookmark} />} title="Bookmark" />,
+    <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+  ]}
+  avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+  heading="Mark Twain"
+  message={(
+    <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+  )}
+  metadata={['Lorem ipsum dolor sit amet', 'Curabitur dolor sapien']}
+/>
+        `
+        ]}
+      />
+
+      <SectionHeading>Without Attribution</SectionHeading>
+      <P>
+        A note can also be used without author attribution. The actions will be
+        moved to the bottom next to the metadata.
+      </P>
+      <Example.React
+        themeToggle
+        includes={{ Note }}
+        codes={[
+          `
+<Note
+  actionBar={[
+    <Note.Action icon={<Icon id={Icon.ids.bookmark} />} title="Bookmark" />,
+    <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+  ]}
+  message={(
+    <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+  )}
+  metadata={['Lorem ipsum dolor sit amet', 'Curabitur dolor sapien']}
+/>
+        `
+        ]}
+      />
+      <SectionHeading>List</SectionHeading>
+      <P>Use a List when display a group of related Notes.</P>
+      <Example.React
+        themeToggle
+        includes={{ Note }}
+        codes={[
+          `
+<Note.List>
+  <Note
+    avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+    heading="Mark Twain"
+    message={(
+      <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+    )}
+  />
+
+  <Note
+    avatar={<Avatar name="Samuel Lanhorne Clemens" src="//placebear.com/128/128" />}
+    heading="Samuel Lanhorne Clemens"
+    message={(
+      <p>There are those who imagine that the unlucky accidents of life—life's 'experiences'—are in some way useful to us. I wish I could find out how. I never know one of them to happen twice. They always change off and swap around and catch you on your inexperienced side.</p>
+    )}
+  />
+
+  <Note
+    avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+    heading="Mark Twain"
+    message={(
+      <p>Clothes make the man. Naked people have little or no influence on society.</p>
+    )}
+  />
+</Note.List>
+`
+        ]}
+      />
+    </Content>
+  </Chrome>
```

Comment:
Getting a few errors in-browser on this page:

```
warning.js?3119d7c:33 Warning: Failed prop type: Invalid prop `avatar` supplied to `Note`.
    in Note
    in ReactExample (at note.js?entry:103)
  


warning.js?3119d7c:33 Warning: Failed prop type: Invalid prop `children.props.children` of element type `Avatar` supplied to AvatarLink, expected element of type `Avatar`
    in AvatarLink
    in ReactExample (at note.js?entry:121)
   


warning.js?3119d7c:33 Warning: Failed prop type: Invalid prop `icon` of element type `Icon` supplied to Note.Action, expected element of type `Icon`
    in Note.Action
    in ReactExample (at note.js?entry:159)
```

Reply:
**fixed**. i'm seeing that the react team doesn't like this type of prop checker - it implies inheritance and is less flexible. however since we use cloneElement and the children have to be of a specific type i don't know what the alternative is